builtin/plugins/dposv3/dpos.go

```diff
@@ -197,6 +198,40 @@ func (c *DPOS) Init(ctx contract.Context, req *InitRequest) error {
 		params.DowntimePeriod = defaultDowntimePeriod
 	}
 
+	candidates, err := LoadCandidateList(ctx)
```

Comment:
Should just be `candidates := &CandidateList{}`

Reply:
Done